code/modules/events/wizard/rpgloot.dm

```diff
@@ -21,7 +23,38 @@
 		else
 			I.name = "[I.name] of [pick(suffixes)]"
 
-		I.force 		+= quality
-		I.force			= max(0,I.force)
-		I.throwforce 	+= quality
-		I.throwforce	= max(0,I.throwforce)
\ No newline at end of file
+		I.force 		= max(0,I.force + quality)
+		I.throwforce	= max(0,I.throwforce + quality)
+		I.armor			+= quality
```

Comment:
The help file told me if you added a number to a list it would just add that number to every entry in the list. To be fair I haven't checked yet to see if that ACTUALLY WORKS. Give me a moment here.